src/collections/Emails.ts

```diff
@@ -0,0 +1,96 @@
+import { isAuthor } from '@/access/isAuthor'
+import { canPublish } from '@/access/canPublish'
+import { isAdmin } from '@/access/isAdmin'
+import type { CollectionConfig, PayloadRequest } from 'payload'
+import { getNextYearDate, toTitleCase } from '@/app/(payload)/utils.ts/collectionUtils'
+import { SUBSCRIPTIONS_LIST } from '@/app/(payload)/constants/subscriptionsList'
+
+export const Emails: CollectionConfig = {
+  slug: 'emails',
+  admin: {
+    useAsTitle: 'title',
+    components: {
+      edit: {
+        PublishButton: '../app/(payload)/components/PublishButton',
+      },
+    },
+  },
+  versions: {
+    drafts: true,
+  },
+  access: {
+    read: ({ req }: { req: PayloadRequest }) => {
+      return isAuthor({ req })
```

Comment:
shouldn't only admins have access to user emails? why would an author need this?